src/ChainSafe.Gaming.Gelato/GelatoClient.cs

```diff
@@ -35,6 +47,11 @@ public GelatoClient(IHttpClient httpClient, GelatoConfig config)
             return (await httpClient.Post<TRequest, TResponse>(url, request)).EnsureResponse();
         }
 
+        /// <summary>
+        /// Retrieves an array of supported networks from the Gelato Relay service.
+        /// </summary>
+        /// <returns>An array of strings representing the supported networks by the Gelato Relay service.</returns>
```

Comment:
it would be nice to describe if those strings are network names, string representation of network IDs or something else.